mozilla/bedrock
# New mofo leader

## One-line summary

This PR adds a new MOFO leader to the leadership page. 

**Note for reviewer**: this change is scheduled to deploy on July 1st. Please don't merge until ready for deployment.

## Significant changes and points to review

`/about/leadership/` MOFO section

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/14744


## Testing
1. Check out this PR
2. Go to http://localhost:8000/en-US/about/leadership/ to review new boi
